lib/hyacinth/dynamic_fields_map.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Hyacinth
+  module DynamicFieldsMap
+    # Generates map of dynamic fields for the given metadata form.
+    #
+    # @param [String|Array<String>] for_metadata_form limits map to fields related forms
+    def self.generate(for_metadata_form)
```

Comment:
would it be easier to use the splat operator here?

Reply:
Yes, added that in.